src/proto/game_state.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto2";
+
+package TbotsProto;
+
+import "proto/geometry.proto";
+import "proto/vision.proto";
+
+message GameState
+{
+    enum PlayState
+    {
+        PlayState_HALT    = 0;
+        PlayState_STOP    = 1;
+        PlayState_SETUP   = 2;
+        PlayState_READY   = 3;
+        PlayState_PLAYING = 4;
+    }
+    enum RestartReason
+    {
+        NONE           = 0;
```

Comment:
I think these should be prefixed with RestartReason_ and similarly for RefereeCommand. @akhilveeraghanta thoughts?

Reply:
Won't these be accessed through the RestartReason namespace?

If it would be `RestartReason::DIRECT` then we don't need the extra `RestartReason_` prefix, but if its just `DIRECT` then prefixing makes sense

Reply:
No, protobuf doesn't make enum classes or namespaces for enums. https://stackoverflow.com/questions/27030332/solutions-to-resolve-enum-field-naming-restriction-in-google-protobuf-due-to-c

Reply:
Ah good to know TIL

From here looks like they say: https://developers.google.com/protocol-buffers/docs/style

> Prefer prefixing enum values instead of surrounding them in an enclosing message. The zero value enum should have the suffix UNSPECIFIED.

But yeah prefixing makes sense! We should add it to our proto style guide if its not there already, we can probably just link to  https://developers.google.com/protocol-buffers/docs/style

Reply:
so RESTART_REASON_NONE, then